dev/io.openliberty.jakarta.jsonb.3.0_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -63,21 +63,20 @@
 		    </exclusions>
 		</dependency>
 		<!-- api - jsonb - internal bundle -->
-		<!-- TODO revert to jakarta.json.bind when GA version is in GHE -->
 		<dependency>
 			<groupId>io.openliberty.jakarta.json.bind</groupId>
 			<artifactId>jakarta.json.bind-api</artifactId>
```

Comment:
So I think the TODO here was to switch `io.openliberty.jakarta.json.bind` to `jakarta.json.bind`